src/borrower/pile.sol

```diff
@@ -59,9 +60,10 @@ contract Pile is DSNote, Auth, Interest {
      // --- Public Debt Methods  ---
     /// increases the debt of a loan by a currencyAmount
     /// a change of the loan debt updates the rate debt and total debt
-    function incDebt(uint loan, uint currencyAmount) external auth note {
+    function incDebt(uint loan, uint currencyAmount) external auth note { 
         uint rate = loanRates[loan];
         require(now == rates[rate].lastUpdated, "rate-group-not-updated");
+        currencyAmount = safeAdd(currencyAmount, rmul(currencyAmount, rates[rate].fixedRate));
```

Comment:
We could safe one addition if the fixed rate is in a different format:

For example 2% fixed fee instead of `0.02` we could express it as `1.02`
and therefore just multiply once

```suggestion
        currencyAmount =  rmul(currencyAmount, rates[rate].fixedRate));
```